styles/markdown.module.css

```diff
@@ -95,12 +95,6 @@
   @apply py-2 ml-6 text-base md:text-lg lg:text-xl;
 }
 
-.markdown li a,
-.markdown h3 a,
-.markdown h2 a {
-  @apply text-green;
-}
-
```

Comment:
???

Reply:
where did that that come from? definitely didn't edit this directly—is it generated and not supposed to be committed?

Reply:
You should retain that css to keep all the links green inside posts — currently inside headers or lists links just have no styling

Reply:
Whoops. Added back.